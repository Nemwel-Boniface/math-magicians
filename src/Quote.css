.quote {
  position: absolute;
  top: 40%;
  width: 800px;
  left: 23%;
}

.quote h2 {
  font-size: 35px;
}