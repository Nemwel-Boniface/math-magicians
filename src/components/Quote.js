const Quote = () => (
  <div className="quote w3-animate-zoom">
    <h2>
      Math Magicians is not about numbers, equations, computations or algorithms, It is about
      understanding - William Paul Thurston
    </h2>
  </div>
);

export default Quote;
