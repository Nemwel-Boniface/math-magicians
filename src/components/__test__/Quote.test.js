import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Quote from '../Quote';

afterEach(cleanup);
it('Should create a Qoutes snapshot', () => {
  const { asFragment } = render(<Quote />);
  expect(asFragment(<Quote />)).toMatchSnapshot();
});
