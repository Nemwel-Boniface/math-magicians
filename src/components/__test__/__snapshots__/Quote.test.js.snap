// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should create a Qoutes snapshot 1`] = `
<DocumentFragment>
  <div
    class="quote w3-animate-zoom"
  >
    <h2>
      Math Magicians is not about numbers, equations, computations or algorithms, It is about understanding - William Paul Thurston
    </h2>
  </div>
</DocumentFragment>
`;
